src/ImageSharp/ImageFrame{TPixel}.cs

```diff
@@ -145,13 +145,8 @@ internal ImageFrame(Configuration configuration, ImageFrame<TPixel> source)
             source.PixelBuffer.FastMemoryGroup.CopyTo(this.PixelBuffer.FastMemoryGroup);
         }
 
-        /// <summary>
-        /// Gets the image pixels. Not private as Buffer2D requires an array in its constructor.
-        /// </summary>
-        internal Buffer2D<TPixel> PixelBuffer { get; private set; }
-
         /// <inheritdoc/>
-        Buffer2D<TPixel> IPixelSource<TPixel>.PixelBuffer => this.PixelBuffer;
+        public Buffer2D<TPixel> PixelBuffer { get; private set; }
```

Comment:
Can `<inheritdoc/>` publicly inherit docs from an internal interface?

Reply:
I honestly have no idea. I think it triggers a copy of the docs though during build rather than reference the original

Reply:
Ok, we check it during doc generation, and fix if it doesn't work.

Reply:
Intellisense seems to be working in consuming project.